src/services/HealthApi.ts

```diff
@@ -1,10 +1,17 @@
-import { Api, ApiRoute } from '@/services/Api'
+import { BaseApi } from '@/services/BaseApi'
 
-export class HealthApi extends Api {
+export class HealthApi extends BaseApi {
+  protected override routePrefix = '/health'
 
-  protected override route: ApiRoute = '/health'
+  public getHealth(): Promise<boolean> {
+    return this.get<boolean>().then(({ data }) => data)
+  }
 
-  public getHealth(): Promise<string> {
-    return this.get('').then(({ data }) => data)
+  public async checkHealth(): Promise<boolean> {
+    try {
+      return await this.getHealth()
+    } catch {
+      return false
+    }
   }
```

Comment:
I'm guessing this existed somewhere else and you moved it into this service? 

```suggestion
  public async isHealthy(): Promise<boolean> {
    try {
      return await this.getHealth()
    } catch {
      return false
    }
  }
```

Reply:
yeah, both cloud and oss defined basically the same function